### PR TITLE
APP-960: New version of Readme.MD

### PR DESCRIPTION
**Pull request by campidelli**
Contains only information about MessageMLv2 and only one event as example.

### CODE REVIEW

**Comment by vincentmg** [comment]
Updated the readme a bit more. Please take a look @campidelli  @ecarrenho-daitan

File: README.md [hunk 1]
```diff
@@ -15,543 +15,145 @@ When using a custom trust store for the Integration Bridge, make sure it also in
   
 In case the Integration Bridge can not access GitHub, webhooks will still work, but only usernames will be displayed on the events.
 
+## What formats and events it supports and what it produces
+As of the current version, one must set the WebHook Content type to application/json, as shown below:
 ![Selecting content type](src/docs/sample/sample_webhook_content_type.png)
 
-## What formats and events it supports and what it produces
-Every integration will receive a message sent in a specific format (depending on the system it ingests) and will usually convert it into an "entity" before it reaches the Symphony platform. It will also, usually, identify the kind of message based on an "event" identifier, which varies based on the third-party system.
+Every integration will receive a [message](#github-json-message-sent-check-it) sent in a specific format (depending on the system it ingests) and will usually convert it into an object called **MessageML** before it reaches the Symphony platform.
```
Comment (robson-sanchez) [inline README.md]: The 'message' link isn't working.

File: README.md [hunk 2]
```diff
@@ -15,543 +15,145 @@ When using a custom trust store for the Integration Bridge, make sure it also in
...
+The **MessageML** basically contains two structures:
+- A HTML [template](#message-ml-template) used to format the message correctly. _It uses [Apache FreeMarker](http://freemarker.org) conventions to parse the message correctly._
+- A [JSON](#message-ml-data-generated-by-the-parsers), contaning the relevant data sent by the webhook.
+
+It will also, usually, identify the kind of message based on an "event" identifier, which varies based on the third-party system. _Note that currently only **application/json** media type is supported by GitHub integration._
 
 You can find more details about entities and the Symphony Message ML format [here](https://github.com/symphonyoss/App-Integrations-Core#the-message-ml-format).
 
-Currently we support the following ten events from GitHub: 
+Currently we support the following ten events from GitHub:
```
Comment (robson-sanchez) [inline README.md:32]: I think we left the event 'Create a branch or tag' in this list (https://developer.github.com/v3/activity/events/types/#createevent)

**Review by robson-sanchez — CHANGES_REQUESTED** [review]
Really good :+1:

Just take a look at the comment regarding the event list.

File: README.md (hunk 1 above)
Comment (campidelli) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (campidelli) [reply README.md:32]: Done.

**Review by robson-sanchez — APPROVED** [review]

**Pull request merged**